api/v1alpha1/common_types.go

```diff
@@ -52,6 +52,13 @@ type ChaosCondition struct {
 	Reason string `json:"reason"`
 }
 
+const (
+	Injecting string = "injecting"
```

Comment:
using a certain type instead of using string directly.

Reply:
ok, i'll modify it.